### DISCUSSION TITLE
Book4 Ch13 Page 12-13 谱分解

### DISCUSSION THREAD

**Discussion by DR-LHX** [Visualize-ML/Book4_Power-of-Matrix#14] [General]
感谢作者的辛苦付出，此套图书很有帮助。

Book 4 矩阵力量，第13章特征值分解，第12-13页关于谱分解，

公式（27）其实是对所有可对角矩阵A都成立，并非只针对对称矩阵。所以此处说，“特别地，如果A为对称矩阵，A的特征值分解可以写成。。。“，容易引起误解。

另外，此部分内容没有证明，如果A为对称阵，则V为正交矩阵。而是直接用此结论，得到的公式（28）到公式（31）。

额外，第23页，第二段文字，”应的几何操作“应为 ”对应的几何操作“，笔误，小问题。

**Comment by Visualize-ML** [comment]
谢谢你的反馈。《矩阵力量》的错误将会在四月份下旬集中修改，欢迎继续批评指正。

Cheers,
James
